### DISCUSSION TITLE
Disable reorderable for some entries ?

### DISCUSSION THREAD

**Discussion by quasiperfect** [filamentphp/filament#10376] [Help] [answered]
Is possible to disable reorderable for some entries maybe with a callback or something like that ?

**Comment by leandrocfe** [comment]
there is a second parameter

```php
public function table(Table $table): Table
{
    return $table
        ->reorderable('sort', auth()->user()->isAdmin());
}
```

**Reply by quasiperfect** [reply]
thanks @leandrocfe but i was thinking more like for some rows in the table

**Reply by leandrocfe** [reply, marked as answer]
Why would you like to reorder some rows? Does it make sense?

**Reply by quasiperfect** [reply]
let me tell you what i had in mind
i have categories that can have parents (self relation). 
i want to have a custom order but the children should only be moved with the other children
i will add custom actions for reordering like move before or after x. i don't think the reorderable is meant for my use case
